code/modules/reagents/chemistry/reagents/admin_reagents.dm

```diff
@@ -56,6 +57,12 @@
 			continue
 		D.cure(0)
 	..()
+	if(M?.dna?.species)
```

Comment:
I don't think M should ever be null. Not sure about dna tho

Reply:
The reagent is scoped to carbon, so I did the second check just to be safe